torchao/prototype/blockwise_fp8/kernels.py

```diff
@@ -277,3 +277,231 @@ def fp8_blockwise_weight_dequant(
     )
     fp8_blockwise_weight_dequant_kernel[grid](x, s, y, M, N, BLOCK_SIZE=block_size)
     return y
+
+
+# original implementation from fbgemm_gpu:
+# https://github.com/pytorch/FBGEMM/blob/b19401e913fcdff536dc097fa3013a0a9d66256e/fbgemm_gpu/experimental/gemm/triton_gemm/fp8_gemm.py#L3091
+def triton_quantize_fp8_block(
```

Comment:
since we have an optional runtime depdence on fbgemm can we just call their kernel directly?

Reply:
Yes that is the desired end state. For now I have tried and have had repeated problems getting it to work so far (`fbgemm-gpu-genai`), e.g. undefined symbols. Maybe because I'm a b200 devgpu right now, not sure yet.